.github/workflows/ateam.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     concurrency: single
     runs-on: ubuntu-latest
     container: quay.io/testing-farm/tmt:1.6.0
-    timeout-minutes: 30
+    timeout-minutes: 90
```

Comment:
Is this still needed. I think that @juanje pushed a hotfix which is supposed to fix the broken CI. Please rebase your PR and revert this change.

Reply:
is it merged already? I increased the timeout since the additional steps I added: uploading the raw,qcow images and importing the snapshot take up some extra time

Reply:
@guy9050 @lioramilbaum No, at the end, the hotfix didn't need the `timeout` change, so there nothing to rebase.
I think it's probably good idea though to increase the `timeout` because those new tasks are going to take some time...